test/e2e/specs/OIDC_spec.js

```diff
@@ -119,27 +124,26 @@ describe('OIDC flows', function () {
       expect(oidcApp.getCodeFromQuery()).not.toBeNull();
     });
 
-    // https://oktainc.atlassian.net/browse/OKTA-246000
-    xit('PKCE login flow', function () {
+    it('PKCE login flow', function () {
       setup({
         baseUrl: '{{{WIDGET_TEST_SERVER}}}',
-        clientId: '{{{WIDGET_CLIENT_ID}}}',
+        clientId: '{{{WIDGET_SPA_CLIENT_ID}}}',
         redirectUri: 'http://localhost:3000/done',
         authParams: {
           display: 'page',
```

Comment:
may as well set `pkce: true` explicitly here, just for clarity this is what we are testing